pkg/asset/config/appliance_config.go

```diff
@@ -265,7 +263,7 @@ func (a *ApplianceConfig) validateImageRegistry() field.ErrorList {
 		cmd := fmt.Sprintf(PodmanPull, swag.StringValue(a.Config.ImageRegistry.URI))
 		logrus.Debugf("Running uri validation cmd: %s", cmd)
 		args := strings.Split(cmd, " ")
-		if _, err := a.executer.Execute(args[0], args[1:]...); err != nil {
+		if _, err := executer.NewExecuter().Execute(args[0], args[1:]...); err != nil {
```

Comment:
this will create a problem for the unit tests we are about to add. we'll need to Mock the executer.

Reply:
We'll need to see in the installer if there's a workaround. It currently can't be in the asset as it breaks the build when using the image...